tests/sanity/test/install/test-installed-files.js

```diff
@@ -39,6 +39,13 @@ describe('verify installed files', function() {
     await sshHelper.executeCommandWithNoError(`test -f ${process.env.ZOWE_ROOT_DIR}/fingerprint/RefRuntimeHash-*.txt`);
   });
 
+  // # verify the checksums of ROOT_DIR, to self-check zowe-verify-authenticity.sh
+  // $binDir/zowe-verify-authenticity.sh # No parameters!
+
+  it('fingerprint should match', async function() {
+    await sshHelper.executeCommandWithNoError(`${process.env.ZOWE_ROOT_DIR}/bin/zowe-verify-authenticity.sh`);
```

Comment:
This isn't the correct place for this test - this file is checking that files were laid out properly. It was be best to add it in the Ansible script as it's an integration test and it would be good to test the output, not just the return code as if the file was blank it would still pass.

Reply:
I repeat - this shouldn't be in this file.